### PR TITLE
docs(configuration): interactive portal to webpack config

### PR DESCRIPTION
**Pull request by dasarianudeep**
Hi Webpack Team

Often I have seen novice Frontend developers find difficult to swiftly write `webpack.config.js` for their projects with appropriate loader/plugin config.  In an attempt to ease that, I could able to develope a interactive portal - https://generatewebpackconfig.netlify.com/ where user can quickly select the `entry, output, loader` settings and it generates the minimal configuration that he could copy and paste. 

Appreciate if you could have a look at my portal and I presume it greatly helps people to have an idea how the configuration varies for different project requirements.

Best
Anudeep

### CODE REVIEW

**Comment by dhruvdutt** [comment]
cc @ev1stensberg 

We already have webpack-cli init for this, right?

File: src/content/configuration/index.md [hunk 1]
```diff
@@ -394,3 +395,9 @@ module.exports = {
   </details>
 }
 ```
+
+T> Want to rapidly generate webpackconfig file for your project requirements with few clicks away.
```
Comment (ev1stensberg) [inline src/content/configuration/index.md]: This should probably not be `webpackconfig` in one word

**Comment by ev1stensberg** [comment]
That's true, but a interactive GUI is nice too :)

**Comment by dasarianudeep** [comment]
@ev1stensberg I guess the CI passed through now.

**Comment by dasarianudeep** [comment]
@ev1stensberg I see this error in the CI log
```
"message":"API rate limit exceeded for 35.193.7.13. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)"
```

Though it is not related to my changes , do you have any why does that happen.

**Comment by dasarianudeep** [comment]
Ooops..ok

**Comment by dasarianudeep** [comment]
@ev1stensberg Is anything to be done from my end?

**Comment by ev1stensberg** [comment]
@dasarianudeep I don't think so, cc @montogeek

**Comment by dasarianudeep** [comment]
@montogeek  Let me know if I could help anyway yo fix the GH issue

**Comment by dasarianudeep** [comment]
@montogeek Do you mind merging this

**Comment by montogeek** [comment]
Please top merging master, our build has limited resources.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**